bitnami/contour/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: contour
 description: Contour Ingress controller for Kubernetes
-version: 1.2.0
+version: 1.2.1
```

Comment:
it would be a minor at least. Don't you think so?

Reply:
Mhhh I am not sure. To be 100% correct, it is a bugfix of the automatical app update from `1.6.x` to `1.7.x` of your bot :smiley: 
https://github.com/bitnami/charts/commit/04732989c72fb1a03aaed8798d0ae24345190475#diff-8d26fcd3a47b928f0e5ca90b7bc1effcR5